app/assets/javascripts/student_profile/student_profile_page.jsx

```diff
@@ -178,13 +180,34 @@ import {cumulativeByMonthFromEvents} from './QuadConverter';
             {this.renderAttendanceColumn()}
             {this.renderInterventionsColumn()}
           </div>
+          {this.renderTransitionNote()}
           <div style={styles.detailsContainer}>
             {this.renderSectionDetails()}
           </div>
         </div>
       );
     },
 
+    renderTransitionNote: function() {
+      const {currentEducator, actions, transitionNotes, requests} = this.props;
+      const labels = currentEducator.labels;
+
+      const isElemCounselor = _.includes(labels, 'k8_counselor');
+      const isHouseMaster = _.includes(labels, 'high_school_house_master');
```

Comment:
Consider moving these to `PerDistrict`: https://github.com/studentinsights/studentinsights/blob/e68ab2fad2b10bf51bb1fc85415af4addff29939/app/assets/javascripts/helpers/PerDistrict.js#L46